src/components/card/Card.tsx

```diff
@@ -1,22 +1,38 @@
-import BookBody from '../../model/Book'
+import { useEffect, useState } from 'react'
+
+import { Book } from '../../model/Book'
 import './Card.css'
+import placeholderImg from '../../assets/placeholder/placeholderForBook.png'
 
-interface Book {
-  book: BookBody
+interface Books {
```

Comment:
This interface is named Books and you are passing one book to the Card

Reply:
I fix this. Now its name is ShowBook.